app/uk/gov/hmrc/pillar2submissionapi/controllers/actions/AuthenticatedIdentifierAction.scala

```diff
@@ -74,7 +74,7 @@ class AuthenticatedIdentifierAction @Inject() (
   }
 
   override protected def transform[A](request: RequestWithPillar2Id[A]): Future[IdentifierRequest[A]] = {
-    implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromRequest(request)
+    given hc: HeaderCarrier = HeaderCarrierConverter.fromRequest(request)
```

Comment:
Only a suggestion, this stuff is very much a matter of taste.

```suggestion
    given HeaderCarrier = HeaderCarrierConverter.fromRequest(request)
```